js/index.js

```diff
@@ -79,15 +74,55 @@ function submitMessage(event) {
     editButton.addEventListener('click', function(){
        newMessage.innerHTML = `<a href="mailto:${email}">${username}</a>
        <textarea> ${message}</textarea>`
+
+       //Creating the secind Submit button
+       const resubmitButton = document.createElement('button');
+       resubmitButton.innerHTML = "Submit";
+       resubmitButton.setAttribute("type", "submit");
+       resubmitButton.setAttribute("class", "resubmit_button");
+       newMessage.appendChild(resubmitButton);
        newMessage.appendChild(removeButton);
-       newMessage.appendChild(editButton); 
+       
+       newMessage.addEventListener('submit', () => {
+       const message = event.target.textarea.value;
+            newMessage.innerHTML = `<a href="mailto:${email}">${username}</a>
+            <span> ${message}</span>`;
+       })
```

Comment:
I'm not sure offhand why this isn't working. I can look into it more on Monday if you haven't found a solution by then.